test/remote/gateways/remote_payflow_express_test.rb

```diff
@@ -51,65 +51,65 @@ def test_set_express_purchase
   def test_setup_authorization_discount_taxes_included_free_shipping
     amount = 2518
     options = {:ip=>'127.0.0.1',
-      :return_url=>'http://localhost:3000/orders/1/8e06ea26f8add7608671d433f13c2193/commit_paypal?buyer_accepts_marketing=true&utm_nooverride=1',
-      :cancel_return_url=>'http://localhost:3000/orders/1/8e06ea26f8add7608671d433f13c2193/commit_paypal?utm_nooverride=1',
-      :customer=>'test6@test.com',
-      :email=>'test6@test.com',
-      :order_id=>'#1092',
-      :currency=>'USD',
-      :subtotal=>2798,
-      :items => [
-        {:name => 'test4',
-          :description => 'test4',
-          :quantity=>2 ,
-          :amount=> 1399 ,
-          :url=>'http://localhost:3000/products/test4'}],
-      :discount=>280,
-      :no_shipping=>true}
+               :return_url=>'http://localhost:3000/orders/1/8e06ea26f8add7608671d433f13c2193/commit_paypal?buyer_accepts_marketing=true&utm_nooverride=1',
```

Comment:
If we're going with this indentation style, it'd probably make sense to make sure `{`s go on their own line as part of the patch; the following lines (and those for the other hashes in this file) end up being _really_ indented when they wouldn't otherwise need to be.